README.md

```diff
@@ -89,6 +89,14 @@ $ sort-package-json "**/package.json" --check --quiet
 $ sort-package-json "**/package.json" --quiet
 ```
 
+### `--stdin` flag
```

Comment:
I'm not particularly precious about this, but it makes me think; would `--file <path>`/`-F <path>` be a better option here -  where one can use `-` to use stdin. This is how `git` does it (`git commit -F my-template` or `echo foo | git commit -F -`).

Reply:
We plan to use `util.parseArgs` https://github.com/keithamus/sort-package-json/pull/283, not sure if it supports that.

Reply:
```
> util.parseArgs({args: ['-f', '-'], options: {file: {type: 'string', short: 'f' }}})
{ values: [Object: null prototype] { file: '-' }, positionals: [] }
```

Looks like it does?